src/test/java/com/shop/frontend/stepdefinitions/Stepdefs.java

```diff
@@ -2,28 +2,36 @@
 
 import com.shop.frontend.pages.CheckoutPage;
 import com.shop.frontend.pages.MainPage;
+import com.shop.frontend.pages.ProductPage;
 import io.cucumber.java.AfterAll;
 import io.cucumber.java.BeforeAll;
+import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 import org.junit.Assert;
+import org.junit.jupiter.api.Assertions;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
 import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
 
+import java.util.List;
 import java.util.concurrent.TimeUnit;
 
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 
 public class Stepdefs {
 
     public static WebDriver driver;
+    public static ProductPage productPage;
 
     // Author: Nicklas
     @BeforeAll
     public static void setUp() {
+        System.setProperty("webdriver.chrome.driver", "/Users/sachinmehta/chromedriver_mac64_latest/chromedriver");  //Author: Priyanka
```

Comment:
Not sure about this row... Perhaps it will break the test runs everywhere but on your computer?